### ISSUE TITLE
endAt

### ISSUE THREAD

**Issue by YuriiDunaev** [flamelink/flamelink#67]
Hey, thanks for the great tool. Are you think about this 
  ```
  endAt(
      value: number | string | boolean | null,
      key?: string
    ): firebase.database.Query;
```
?
Right now I can path only one param in your API(https://github.com/flamelink/flamelink/blob/master/docs/api-overview.md). 
And the second question. How about firestore database? Is it, the same methods API from the documentation? @jperasmus

**Comment by jperasmus** [comment]
Hi @YuriiDunaev, thanks for the kind words.

We have a newer JavaScript SDK that supports both the Firebase Realtime DB as well as Cloud Firestore. This current/older SDK only supports the Realtime DB.

Cloud Firestore's querying capabilities are much more powerful than the realtime db, so gives you the ability to add as many filters as you'd like.

If you want to look at the newer SDK, you can see the migration guid [here](https://flamelink.github.io/flamelink-js-sdk/#/migration-guide).

Can you please elaborate a bit regarding your first question? I just want to make sure I fully understand what you are asking?

**Comment by YuriiDunaev** [comment]
In your API filtering have 1 parameter for startAt|endAt, but in https://firebase.google.com/docs/reference/js/firebase.database.Query have the second parameter **key**

**Comment by jperasmus** [comment]
Ah okay, I see what you mean now. I wasn't aware of the optional `key` property. What we can do, that is similar to how we handle similar things for Cloud Firestore, is allow either a single value or an array of values, so that if you want an optional key you would specify it like this:

```
app.content.get('schemaKey', {
  orderByChild: 'someField',
  endAt: ['value', 'optionalKey'] // instead of `endAt: 'value'`
})
```

Would that work for you?

**Comment by YuriiDunaev** [comment]
Nope
`this.$flamelink.content
        .get('news', {
          orderByChild: 'date',
          limitToLast: limitToLast + 1,
          endAt: [lastChildDate, lastChild]
        })`
`Error: Query: First argument passed to startAt(), endAt(), or equalTo() cannot be an object.`

**Comment by YuriiDunaev** [comment]
```this.$flamelink.content
        .ref('news')
        .orderByChild('date')
        .limitToLast(limitToLast + 1)
        .endAt(lastChildDate, lastChild)``` this work for me

**Comment by jperasmus** [comment]
Apologies, I did not mean it will work right now. I was thinking we could build that functionality into the SDK, so that the syntax would look like that.

This:
```
this.$flamelink.content
  .get('news', {
    orderByChild: 'date',
    limitToLast: limitToLast + 1,
    endAt: [lastChildDate, lastChild]
  })
```

would basically become this:
```
firebaseApp
  .ref('flamelink/environments/production/content/news/en')
  .orderByChild('date')
  .limitToLast(limitToLast + 1)
  .endAt(lastChildDate, lastChild)
```

**Comment by jperasmus** [comment]
Hi @YuriiDunaev this functionality should now be available as `v0.19.14`.
